### PR TITLE
Feature/25/add mine button

### PR DESCRIPTION
**Pull request by dct-lau17**

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -3,9 +3,13 @@
 require 'json'
 require 'sinatra/base'
 require './lib/blockchain'
+require 'sinatra'
+require 'sinatra/flash'
 
 # Understands sending and receiving transaction hashes
 class BlockchainApp < Sinatra::Base
+  enable :sessions
```
Comment (archmagos) [inline app.rb]: Did you end up needing to use this?

File: app.rb [hunk 2]
```diff
@@ -3,9 +3,13 @@
 require 'json'
 require 'sinatra/base'
 require './lib/blockchain'
+require 'sinatra'
```
Comment (archmagos) [inline app.rb]: Don't think this is needed as we're already requiring Sinatra on line 4

File: app.rb (hunk 1 above)
Comment (dct-lau17) [reply app.rb]: this is needed for the flash messages to appear

Comment (archmagos) [reply app.rb]: Awesome

**Pull request merged**